env0/resource_environment.go

```diff
@@ -195,33 +195,23 @@ func resourceEnvironment() *schema.Resource {
 	}
 }
 
-func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment, configurationVariables client.ConfigurationChanges) {
-	d.Set("id", environment.Id)
-	d.Set("name", environment.Name)
-	d.Set("project_id", environment.ProjectId)
-	safeSet(d, "workspace", environment.WorkspaceName)
-	safeSet(d, "auto_deploy_by_custom_glob", environment.AutoDeployByCustomGlob)
-	safeSet(d, "ttl", environment.LifespanEndAt)
-	safeSet(d, "terragrunt_working_directory", environment.TerragruntWorkingDirectory)
-	safeSet(d, "vcs_commands_alias", environment.VcsCommandsAlias)
+func setEnvironmentSchema(d *schema.ResourceData, environment client.Environment, configurationVariables client.ConfigurationChanges) error {
```

Comment:
There's no error handling if setEnvironmentSchema fails.
I'll take care of it in future iterations.